yb-voyager/cmd/constants.go

```diff
@@ -242,10 +242,11 @@ const (
 var supportedSourceDBTypes = []string{ORACLE, MYSQL, POSTGRESQL, YUGABYTEDB}
 var validExportTypes = []string{SNAPSHOT_ONLY, CHANGES_ONLY, SNAPSHOT_AND_CHANGES}
 
-var validSSLModes = map[string][]string{
-	"mysql":      {"disable", "prefer", "require", "verify-ca", "verify-full"},
-	"postgresql": {"disable", "allow", "prefer", "require", "verify-ca", "verify-full"},
-	"yugabytedb": {"disable", "allow", "prefer", "require", "verify-ca", "verify-full"},
+var AllSSLModes = []string{constants.DISABLE, constants.PREFER, constants.ALLOW, constants.REQUIRE, constants.VERIFY_CA, constants.VERIFY_FULL}
+var ValidSSLModesForSourceDB = map[string][]string{
+	MYSQL:      utils.SetDifference(AllSSLModes, []string{constants.ALLOW}), // MySQL does not support ALLOW mode
```

Comment:
nit: (or rather just a general discussion on code): 

While doing `allSSLModes - allow` is fine; I'd recommend explicitly listing out all the modes. Let's say we need to support a new ssl mode for pg/yb in the future; we will go add it to `AllSSLModes`, but the side effect of that is that it will get added to the MySQL list as well. That makes it slightly error-prone. 

It's also arguably more readable to just read all the modes in a list rather than reading some set logic :) 

To be clear, it's a minor point, but worth thinking about, just in the broader context of writing clean code. See Zen of Python - https://peps.python.org/pep-0020/. One of the core principles of golang as well is "simplicity".

```
Explicit is better than implicit.
Simple is better than complex.
```